tests/login_test.ts

```diff
@@ -1,5 +1,5 @@
 import User from "../helpers/interfaces";
-import LoginPage from "../pages/loginPage";
+import LoginPage = require("../pages/loginPage");
```

Comment:
is that a mix of module import by commonjs and ES?
`import LoginPage from "../pages/loginPage";` should work

Reply:
it does